### PR TITLE
feat: add `/ai` page

### PR DESCRIPTION
**Pull request by donmckenna**
Add a new `/ai` marketing page.
Uses some home page styles and overwrites others.
Adds a new "AI-prompt-code-result-to-live-code" image.

image

### CODE REVIEW

**Review by kc0tlh — CHANGES_REQUESTED** [review]
LGTM, just add "Artificial Intelligence" to the nav bar
image

File: docs/.vitepress/config.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export default defineConfig({
     nav: [
       { text: 'Guides', link: '/guides/introduction' },
       { text: 'Tutorial', link: '/tutorial/1-build-your-first-webcontainer-app' },
-      { text: 'API Reference', link: '/api' },
+      { text: 'Artifical Intelligence', link: '/ai' },
```
Comment (peter-zawistowicz) [inline docs/.vitepress/config.ts]:
```suggestion
      { text: 'Artificial Intelligence', link: '/ai' },
```

File: docs/.vitepress/theme/components/Ai.vue [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+  <div class="top-section">
+    <HeroAi />
+  </div>
+
+  <div class="final-section">
+    <div class="section">
+      <div class="features-small">
+        <div class="feature">
+          <h2>Improve the quality of your AI-generated code</h2>
+          <p>To ensure code generated by your LLM is valid, performant, and satisfies the prompt, you need to be able to actually execute it. WebContainer API enables this securely, right inside the user's browser tab.</p>
```
Comment (peter-zawistowicz) [inline docs/.vitepress/theme/components/Ai.vue:27]:
```suggestion
          <p>To ensure code generated by your LLM is valid, performant, and satisfies the prompt, you need to be able to actually execute it. The WebContainer API enables this securely, right inside the user's browser tab.</p>
```

File: docs/.vitepress/theme/components/Ai.vue [hunk 3]
```diff
@@ -0,0 +1,240 @@
...
+        </div>
+        <div class="feature">
+          <h2>Provide a better developer experience for your users</h2>
+          <p>With WebContainer API you can provide your users a low-latency in-browser code execution environment seamlessly integrated into your AI application so they can get production-ready code faster.</p>
```
Comment (peter-zawistowicz) [inline docs/.vitepress/theme/components/Ai.vue:31]:
```suggestion
          <p>With the WebContainer API you can provide your users a low-latency in-browser code execution environment seamlessly integrated into your AI application so they can get production-ready code faster.</p>
```

File: docs/.vitepress/theme/components/Ai.vue [hunk 4]
```diff
@@ -0,0 +1,240 @@
...
+          <h2>Avoid infrastructure and security headaches</h2>
+          <p>With no virtual machines or network connections to manage, you can ensure untrusted AI-generated code doesn’t pose security risks to you or your end users.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="section">
+    <h2>Leverage the tech <strong>we use<br/> in our own products.</strong></h2>
+    <p>WebContainer API already powers production developer experiences from interactive tutorials to full-blown IDEs. With the WebContainer API, you can build a secure and performant coding environment into your AI applications without worrying about provisioning infrastructure.</p>
```
Comment (peter-zawistowicz) [inline docs/.vitepress/theme/components/Ai.vue:43]:
```suggestion
    <p>WebContainers already powers production developer experiences from interactive tutorials to full-blown IDEs. With the WebContainer API, you can build a secure and performant coding environment into your AI applications without worrying about provisioning infrastructure.</p>
```

**Review by peter-zawistowicz — CHANGES_REQUESTED** [review]

File: docs/.vitepress/config.ts (hunk 1 above)
Comment (donmckenna) [reply docs/.vitepress/config.ts]: already corrected

**Review by peter-zawistowicz — APPROVED** [review]
🚀

**Pull request merged**